docs/README.md

```diff
@@ -2,32 +2,55 @@
 navTitle: Documentation
 ---
 
-# FlowForge Docs
+# Welcome to FlowForge
 
 FlowForge helps Node-RED developers deliver applications in a more reliable,
 collaborative, and secure manner. Node-RED’s intuitive, low-code development
 environment is great for connecting together hardware devices, APIs, and online
-services. FlowForge adds to Node-RED; collaborative development, management of
+services.
+
+FlowForge adds to Node-RED; collaborative development, management of
 remote deployments, support for DevOps delivery pipelines, and the ability to
 host Node-RED applications on FlowForge. FlowForge is the DevOps platform for
 Node-RED application development and delivery.
 
-## Overview
+## Using FlowForge
 
- - [FlowForge Concepts](./user/concepts.md) - understanding the platform
- - [FlowForge Cloud](./cloud/) - details specific to the hosted service
- - [Administrator documentation](./admin/) - how to administer the platform
- - [User documentation](./user/) - how to use the platform
+ - [FlowForge Concepts](./user/concepts.md) - Understanding the core terminology and concepts that make up the FlowForge platform.
```

Comment:
Styling wise, what do you think about using tables with bigger cells? I've tried this in https://deploy-preview-935--flowforge-website.netlify.app/handbook/ and found it appealing.

Reply:
I really liked it. I held off on this, as I had already started the work by the time I saw it. Happy to do that tabling as a follow up PR once we've settled on ordering/grouping here.